pvlib/pvsystem.py

```diff
@@ -1835,6 +1836,76 @@ def _pwr_optfcn(df, loc):
     return I * df[loc]
 
 
+def sdm_sum_current(resistance_shunt, resistance_series, nNsVth, current,
```

Comment:
In the spirit of functions that start with verbs and of avoiding most abbreviations, I think the name should be ``sum_singlediode_current``

Reply:
This needs an api.rst entry.